docs/source/reference/numpysupported.rst

```diff
@@ -582,6 +582,42 @@ Modules
 ``random``
 ----------
 
+Generator Objects
+'''''''''''''''''
+Numba supports ``numpy.random.Generator()`` objects. Users can now pass
```

Comment:
If this is processed by Sphinx, then double backticks will lack the formatting (and sometimes linking) provided by single ticks and a domain/role specification.
E.g.
```suggestion
Numba supports :py:class:`numpy.random.Generator` objects. Users can now pass
```